doc_templates/phylum_org_link.md

```diff
@@ -0,0 +1,10 @@
+{PH-HEADER}
+
+{PH-MARKDOWN}
+
+## Examples
+
+```sh
+# Set `sample` as the default organization for all future operations
+$ phylum org link sample
```

Comment:
```suggestion
# Interactively select an organization for all future operations
$ phylum org link

# Set `sample` as the default organization for all future operations
$ phylum org link sample
```
This is just a suggestion for making this example look more like the one in `doc_templates/phylum_init.md`. The same change _could_ be made for each of the other `org` commands that allows for interactively selecting the organization, but that is likely too much. `phylum org link` feels different, though, in this respect.